passenger-service/cab-aggreagator-passenger-service/passenger-service/src/main/java/com/modsen/passenger/controller/PassengerController.java

```diff
@@ -0,0 +1,51 @@
+package com.modsen.passenger.controller;
+
+import com.modsen.passenger.dto.PassengerRequestDto;
+import com.modsen.passenger.dto.PassengerResponseDto;
+import com.modsen.passenger.dto.PageResponseDto;
+import com.modsen.passenger.dto.PassengerListResponseDto;
+import com.modsen.passenger.service.PassengerService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("api/v1/passengers")
+@RequiredArgsConstructor
+public class PassengerController {
+    private final PassengerService passengerService;
+
+    @PostMapping
+    public ResponseEntity<PassengerResponseDto> savePassenger(@RequestBody PassengerRequestDto passengerRequestDto) {
+        PassengerResponseDto savedPassengerDto = passengerService.savePassenger(passengerRequestDto);
+        return ResponseEntity.status(HttpStatus.CREATED).body(savedPassengerDto);
```

Comment:
Обычно в таких случаях переносят на новую строку методы
```suggestion
        return ResponseEntity.status(HttpStatus.CREATED)
        .body(savedPassengerDto);
```